coordinator/src/routes/type-guards.ts

```diff
@@ -0,0 +1,31 @@
+import {fhir} from "@models"
```

Comment:
I like this refactor, nicely decoupled from hapi now, but is `routes/` the right place for this now?